Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8-buster
+WORKDIR /home/raw
+
+# install psql
+# see https://www.postgresql.org/download/linux/ubuntu/
+RUN apt-get update && apt-get install -y lsb-release && \
+    sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list' && \
+    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
+    apt-get update && \
+    apt-get -y install postgresql-client-13
+
+# raw
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+COPY raw ./raw
+
+# tests
+COPY tests ./tests
+RUN pip install -r ./tests/requirements.txt
```

Comment:
As much as I like both Docker and PostgreSQL, they seem like more technology than we really need to introduce here. As our database connections are handled by SQLAlchemy, and we don't need to test its ability to connect to various database systems, I think SQLite is sufficient for testing purposes. As I understand it, all the Docker machinery is here mainly to provide a temporary Postgres instance to connect to, but I think we can do without that and just use SQLite instead.

(As an aside, for cases where you really might need to test against Postgres, I recommend looking at https://github.com/CloverHealth/pytest-pgsql which gets you there without having to bring Docker into it.)